frontend/src/app/services/validate.ts

```diff
@@ -19,6 +19,24 @@ function upToEpsilonCustomizer(value1: any, value2: any) {
   }
 }
 
+function upToTwoCustomizer(value1: any, value2: any) {
+  if (isNumber(value1) && isNumber(value2)) {
+    return Math.abs(value1 - value2) <= 2;
+  }
+}
+
+function gameDataIsEqualCustomizer(value1: any, value2: any) {
+  if (value1.total === undefined || value1.totalUnrounded === undefined || value1.avg7Day === undefined) {
+    return upToEpsilonCustomizer(value1, value2);
+  }
+
+  // Special handling of game statistics
+  return isEqualWith(value1, value2, upToTwoCustomizer)
```

Comment:
Why 2, what kind of magic constant is that? Pls, explain in the comment at least.